### PR TITLE
Test

### PR DESCRIPTION
**Pull request by 95RahulKumar**
please accept changes

### CODE REVIEW

File: src/app/components/contact-us/contact-us.component.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class ContactUsComponent implements OnInit, AfterViewInit {
   listofItvs: Array<string> = ['ITV01', 'ITV02', 'ITV03', 'ITV04', 'ITV05'];
   // data
   public lineChartData: Array<number> = [1, 8, 49];
-  // testing push for auther also
```
Comment (95RahulKumar) [inline src/app/components/contact-us/contact-us.component.ts]: Change

**Review by 95RahulKumar — COMMENTED** [review]
done

**Review by 95RahulKumar — COMMENTED** [review]
done

**Review by 95RahulKumar — COMMENTED** [review]
done

**Review by Joyel42 — APPROVED** [review]

**Review by Joyel42 — APPROVED** [review]

**Review by Joyel42 — APPROVED** [review]

**Comment by Joyel42** [comment]
Try now

**Review by Joyel42 — APPROVED** [review]

**Pull request merged**